jest.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
   ],
   preset: "ts-jest",
   testEnvironment: "node",
-  testTimeout: 10000,
+  testTimeout: 25000,
```

Comment:
Why did this need to increase? Will test potentially now take longer to run?

Reply:
Because tests take between 20 and 30 seconds to pass
10 seconds is too short